starter-project-web-vue2/components/amanabiy/AmanBlog.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+
+  <v-list
+    flat
+    subheader
+    three-line
+    class="my-50"
+  >
+    <v-card class="my-20" width="90%">
+      <v-subheader> Blogs </v-subheader>
+      <div v-for="blog in blogs" :key="blog.id"> 
+        <v-list-item  :class="{'grey lighten-4': blog.id % 2 && !blog.done}">
+            <v-list-item-action>
+            </v-list-item-action>
+
+            <v-list-item-content>
+                <nuxt-link
+                  style="text-decoration: none; color: inherit"
+                  :to="'/amanabiy/' + blog.id"
```

Comment:
nit; [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) is more readable